docs/TimerMocks.md

```diff
@@ -40,7 +40,10 @@ test('waits 1 second before ending the game', () => {
 ```
 
 Here we enable fake timers by calling `jest.useFakeTimers();`. This mocks out
-setTimeout and other timer functions with mock functions.
+setTimeout and other timer functions with mock functions. If running multiple
+tests inside of one file or describe block, `jest.useFakeTimers();` should be
```

Comment:
What do you think about replacing "should" with "can"?I think whether you should do this or not depends on how you're testing